rdt/hyper_transformer.py

```diff
@@ -212,6 +212,44 @@ def update_default_data_type_transformers(self, new_data_type_transformers):
         self.default_data_type_transformers.update(new_data_type_transformers)
         self._unfit()
 
+    def update_transformers(self, column_name_transformer):
+        """Update any of the transformers assigned to each of the column names.
+
+        Args:
+            column_name_transformer(dict):
+                Dict mapping column names to transformers to be used for that column.
+        """
+        if self._fitted:
+            warning_msg = (
+                "Warning: For this change to take effect, please refit your data using 'fit' "
+                "or 'fit_transform'."
+            )
+            print(warning_msg)  # noqa: T001
```

Comment:
should we create custom tip and warning methods that just print the text with the label so that we can just ignore the lint problem in one place? eg:
```
def tip(text):
    print(f'Tip: {text}') # noqa:T001
````